SupportClient/SupportClient/Controllers/ViewController.swift

```diff
@@ -0,0 +1,210 @@
+        newConversationButton.layer.cornerRadius = 22
+        view.addSubview(newConversationButton)
+        
+    }
+    
+    func setupFeedbackTableView() {
+        
+        feedbackTableView.delegate = self
+        feedbackTableView.dataSource = self
+        feedbackTableView.register(FeedbackTableViewCell.self, forCellReuseIdentifier: FeedbackTableViewCell.reuseID)
+        view.addSubview(feedbackTableView)
+                    
+    }
+    
+    func setupConstraints() {
+        if !feedbackData.isEmpty {
+            feedbackTableView.snp.makeConstraints{ make -> Void in
+                make.leading.trailing.top.bottom.equalToSuperview()
+            }
+        } else {
+            feedbackTableView.isHidden = true
+            newConversationButton.snp.makeConstraints{ make -> Void in
+                make.centerX.equalToSuperview()
+                make.bottom.equalToSuperview().inset(50)
+                make.width.equalTo(view.frame.width / 2)
+                make.height.equalTo(45)
+            }
+            feedbackLabel.snp.makeConstraints{ (make) -> Void in
+                make.centerX.equalToSuperview()
+                make.bottom.equalToSuperview().inset(view.frame.height / 2)
+            }
+        }
+    }
+    
+    @objc func handleNavigationBarRightTap() {
+        
+        let isEvenNumTaps = countEditTaps % 2 == 0
+        feedbackTableView.setEditing(isEvenNumTaps, animated: true)
+        countEditTaps += 1
+
+    }
+    
+}
+
+
+extension ViewController: UITableViewDelegate {
+    
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return 75
+    }
+    
+    
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        let cell = feedbackTableView.cellForRow(at: indexPath) as! FeedbackTableViewCell
+        var feedback = feedbackData[indexPath.row]
+        feedback.hasRead.toggle() // TODO: remove this once hooked up to database
+        cell.toggleRead(for: feedback) // TODO: remove this once hooked up to database
+    }
+    
+    func tableView(_ tableView: UITableView, commit editingStyle: UITableViewCell.EditingStyle, forRowAt indexPath: IndexPath) {
+        if editingStyle == .delete {
+            feedbackData.remove(at: indexPath.row)
+            feedbackTableView.deleteRows(at: [indexPath], with: .fade)
+        }
+        // If all conversations removed, go back to default page
+        if feedbackData.isEmpty {
+            view.snp.removeConstraints()
+            feedbackTableView.isHidden = true
+            newConversationButton.snp.makeConstraints{ make -> Void in
+                make.centerX.equalToSuperview()
+                make.bottom.equalToSuperview().inset(50)
+                make.width.equalTo(view.frame.width / 2)
+                make.height.equalTo(45)
+            }
+            feedbackLabel.snp.makeConstraints{ (make) -> Void in
```

Comment:
instead of removing constraints on `view`, you can just do `remakeConstraints` on line 180 and 186 which will remove all their old constraints and make these new ones